recipes/h5py/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "2.6.0" %}
+
+package:
+  name: h5py
+  version: {{ version }}
+
+source:
+  fn: h5py-{{ version }}.tar.gz
+  url: https://github.com/h5py/h5py/archive/{{ version }}.tar.gz
+  md5: 613c5776192eaf02ff96a82dfc6d2e78
+  patches:
+    # Patch a test that appears to be failing in some cases.
+    # This patch has been submitted to upstream.
+    # See the linked PR for more information.
+    #
+    # https://github.com/h5py/h5py/pull/697
+    #
+    - h5py_tests_old_test_file.py.patch
```

Comment:
Had to add this patch to skip a flaky part of a test. This was submitted upstream in this PR ( https://github.com/h5py/h5py/pull/697 ).